### PR TITLE
Merging Branch Dev into Main after Finalization

### PR DESCRIPTION
**Pull request by rebaz36**
Hey Reviewer This is @rebaz36 & ✅  @ndorrh 
We've followed the project requirements to the best of our knowledge and have finished everything mentioned below

- #25 ✅  @rebaz36
- #24 ✅  @rebaz36
- #23 ✅  @rebaz36
- #22 ✅  @rebaz36
- #21 ✅  @rebaz36
- #20 ✅  @rebaz36
- #18 ✅  @rebaz36
- #17 ✅  @ndorrh 
- #15 ✅  @rebaz36
- #13 ✅  @ndorrh 
- #12 ✅  @ndorrh 
- #10 ✅  @rebaz36
- #9 ✅  @rebaz36
- #8 ✅  @ndorrh 
- #6 ✅  @rebaz36
- #5 ✅  @ndorrh 
- #3 ✅  @ndorrh 
- #2 ✅  @rebaz36 

## Besides this we've also made some design tweaks but also ensured to stick to the main design as much as possible, View our project Live [here](https://rebaz36.github.io/SpaceTravelers-React_Redux/)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,37 @@
...
-        >
-          Learn React
-        </a>
-      </header>
+      <NavBar />
+      <Routes>
+        <Route
+          path="SpaceTravelers-React_Redux/rockets"
+          element={<Rockets />}
+        />
```
Comment (Kamba56) [inline src/App.js:25]: - [ ] As demanded by the project requirements, kindly make the rockets page the default view when the app first loads.

File: src/Components/NavBar.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        <h1>|</h1>
+        <NavLink
+          className="Nav-Item"
+          to="SpaceTravelers-React_Redux/profile"
+          style={({ isActive }) => (isActive ? { color: 'red' } : { color: 'blue' })}
+        >
+          My Profile
+        </NavLink>
+      </div>
+    </div>
```
Comment (Kamba56) [inline src/Components/NavBar.jsx:37]: - [ ] Please make sure that only the active navlink is underlined. I can see that all the navlinks are underlined in your work. Kindly fix this. please.

File: src/App.js [hunk 3]
```diff
@@ -1,25 +1,37 @@
...
+import Missions from './Pages/Missions';
+import Profile from './Pages/Profile';
+
 import './App.css';
 
 function App() {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(showMissions());
+  }, []);
```
Comment (Kamba56) [inline src/App.js:17]: - [ ] As demanded by the project requirements, the missions data should only be fetched (once) when a user navigates to the Missions section. I can see that you are fetching the data when the App is rendered. Kindly fix this please.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @rebaz36, @ndorrh, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great use of Github workflow!👏👏👏
- [x] All linters passed

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (rebaz36) [reply src/App.js:23]: Thanks for the note @Kamba56 Fixed this

File: src/Components/NavBar.jsx (hunk 2 above)
Comment (rebaz36) [reply src/Components/NavBar.jsx]: Solved ✅

File: src/App.js (hunk 3 above)
Comment (rebaz36) [reply src/App.js]: we've Fixed this as well 🍾

**Review by emmiiorji — APPROVED** [review]

File: src/Components/NavBar.css [hunk 4]
```diff
@@ -0,0 +1,39 @@
+/* stylelint-disable selector-type-case */
+/* stylelint-disable selector-type-no-unknown */
```
Comment (emmiiorji) [inline src/Components/NavBar.css:2]: [Optional] Kindly avoid disabling inline linter checks as they help you maintain best coding practices and share coding standards with your team.

**Review by emmiiorji — APPROVED** [review]
Hi Team, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Good Highlights✔

- Good design implementation
- Linter checks are passing
- Good commit messages
- Requested changes have been implemented


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/NavBar.css (hunk 4 above)
Comment (ndorrh) [reply src/Components/NavBar.css:2]: Thanks for the review.
We will take that into consideration

**Pull request merged**